prime-router/examples/upload-synthetic-data/readme.md

```diff
@@ -0,0 +1,80 @@
+# Uploading Synthetic Data
+
+1. Compile and Run PRIME Data Hub
+2. Generate synthetic CSV data
+3. Post CSV data to PRIME Web API
+4. Examine
+
+### Compile and Run PRIME Data Hub
+
+```bash
+# from the `prime-router/` directory
+
+# run infrastructure like PostgreSQL 
+bash devenv-infrastructure.sh
+
+# build and run PRIME Data Hub
+docker-compose up -d
+
+# optionally, follow logs of `prime_dev` container
+docker-compose logs -f prime_dev
+```
+
+### Generate synthetic data
+
+Use `prime data` command to generate synthetic data.
```

Comment:
```suggestion
Use `prime data` command to generate fake data.
```

I think we are distinguishing between 'fake' data and Mo's new 'synthetic' data generator, which at this moment is still in his PR branch #404.

This command as written generates fake data, so we should probably label it that way. (I think the intention is that Mo's --synthesize command is you start with real data, and transmogrify it.)

Reply:
I didn't understand that distinction and the nuance between them -- thank you! I've updated the folder name and all other references of _synthetic_ to _fake_ as well. 

Another example might be worthwhile for me to explain the difference.

Reply:
Fixed in 3da3651d8d19f269cab03a6bac461b958660827d